src/main/java/mate/academy/bookshop/specification/book/BookSpecificationBuilderImpl.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.bookshop.specification.book;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookshop.dto.BookSearchParameters;
+import mate.academy.bookshop.model.Book;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class BookSpecificationBuilderImpl implements BookSpecificationBuilder {
+    private static final String AUTHOR_COLUMN_NAME = "author";
+    private static final String TITLE_COLUMN_NAME = "title";
+    private static final String PRICE_COLUMN_NAME = "price";

```

Comment:
we can import constants from their respective `PriceSpecificationProvider` for example and other classes instead of duplicating them